doc_admin/AdminManual.adoc

```diff
@@ -79,26 +79,22 @@ Open Hospital official releases can be downloaded from GitHub: https://github.co
 
 More information can be found on the Open Hospital website: https://www.open-hospital.org/download
 
-[#setup]
-=== 1.6 Initial setup
+[#unpack]
+=== 1.6 Unpack
 
-Open Hospital does not require installation. It is sufficient to unzip the downloaded package in a folder of your choice, e.g.:
+NOTE: Do not double-click on the **oh.bat** startup script without unzipping the package first!
 
-* on Linux:
-----
-/home/OH/OpenHospital-[VERSION]
-----
+Most operating systems allow to "open" a downloaded compressed archive by just double-clicking on it. 
+In this way, the archive is just opened, but not uncompressed. Let's say it is open for "inspection" only. 
 
-* on Windows:
-----
-C:\Users\OH\OpenHospital-[VERSION]
-----
+In order to work with the application you need to unzip (uncompress) the downloaded package in a folder of your choice.
 
-To uncompress the package:
+To uncompress the package: *Right-click on the package -> Extract all/here*
+
+*Alternative*: to uncompress with command line instructions
```

Comment:
add a colon (:) at the end of the line

Reply:
two subsquent colons? line 94 is meant to be more like a title (but not indented like other titles)

Reply:
Ok, I would write "Alternative method: uncompress package from command line"